compiler/optimizer/Liveness.cpp

```diff
@@ -70,21 +70,27 @@ TR_Liveness::TR_Liveness(TR::Compilation           *comp,
      _liveVariableInfo(liveVariableInfo)
    {
    _traceLiveness = comp->getOption(TR_TraceLiveness);
-   if (traceLiveness())
-      traceMsg(comp, "Starting Liveness analysis\n");
 
    if (liveVariableInfo == NULL)
       // can be re-used by the caller because it's allocated in caller's stack
       _liveVariableInfo = new (trStackMemory()) TR_LiveVariableInformation(comp, optimizer, rootStructure, splitLongs, includeParms,
                                                                            ignoreOSRUses);
    else
       _liveVariableInfo = liveVariableInfo;
+   }
+
+void TR_Liveness::runLiveness(TR_Structure *rootStructure)
```

Comment:
Perhaps simply calling this `perform()` is consistent with other optimizer analyses.

Reply:
Changed to perform() (and fixed the missing copyright update) in the latest push.